src/router/index.js

```diff
@@ -66,4 +59,36 @@ export const routes = [
     },
     component: Events,
   },
+  {
+    path: "/admin",
+    name: "Admin Stuff",
+    component: Admin,
+    meta: {
+      admin: true
+    }
+  },
+  {
+    path: "/redirects",
+    name: "Redirects", 
+    component: Redirects,
+    meta: {
+      admin: true
+    }
+  },
+  {
+    path: "/edit-staff",
```

Comment:
Super minor, but personally I'd recommend `/admin/staff` and `/admin/events` since they're both part of the `/admin` interface.